pom.xml

```diff
@@ -242,11 +244,12 @@
         <configuration>
           <charset>UTF-8</charset>
           <doclint />
-          <serialwarn>false</serialwarn><!-- Serial annotation is introduced with Java 14. -->
-          <failOnError>true</failOnError>
-          <failOnWarnings>true</failOnWarnings>
+          <serialwarn>false</serialwarn><!- - Serial annotation is introduced with Java 14. - ->
+          <failOnError>false</failOnError>
+          <failOnWarnings>false</failOnWarnings>
         </configuration>
       </plugin>
+      -->
```

Comment:
You can remove these lines instead of commenting.
Is there reason for commenting them?

Reply:
wait... that should never have been committed in the first place!